cmake/bintools/gnu/target_objdump.cmake

```diff
@@ -19,7 +19,7 @@ function(bintools_objdump)
     # Prefix of output variables
     BINTOOLS_OBJDUMP
     # List of argument names without values, hence boolean
-    "DISASSEMBLE;DISASSEMBLE_SOURCE"
+    "DISASSEMBLE;DISASSEMBLE_SOURCE;DISASSEMBLE_ALL"
```

Comment:
You forgot to add `DISASSEMBLE_ALL` to description of the function `bintools_objdump` above ^^^

Reply:
Done!